src/main/java/se/sundsvall/smloader/integration/db/model/CaseMapping.java

```diff
@@ -30,16 +30,13 @@ public class CaseMapping {
 	private String externalCaseId;
 
 	@Id
-	@Column(name = "caseId")
-	private String caseId;
+	@Column(name = "errandId")
```

Comment:
Should probably be "errand_id"

Reply:
Noticed there is multiple coliumns with camelCase, not sure if it is supposed to be like that or not.